.github/workflows/update-project-field.yml

```diff
@@ -0,0 +1,71 @@
+name: Update Project Field to Match Label
+
+# Triggers the workflow when a label is added to an issue.
+on:
+  issues:
+    types:
+      - labeled
+
+# Details needed to update a project field.
+env:
+  ORGANIZATION: "Automattic"
+  PROJECT_NUMBER: 391
+  PRIORITY_PREFIX: "[Pri]"
+  PRIORITY_FIELD: "Priority"
+
+jobs:
+
+  # Updates project fields to match label changes.
+  update_project_field: 
+    runs-on: ubuntu-latest
+    steps:
+
+      # Checks if the issue is already in the project. If not, the next steps will be skipped.
+      - name: Check if issue is in project
+        env:
+          GITHUB_TOKEN: ${{ secrets.PROJECT_AUTOMATION_TOKEN }}
```

Comment:
> I see a secret configured in wp-calypso secrets called PROJECT_AUTOMATION_TOKEN

Thanks again for discovering this! I removed the token generation step and replaced it with the token you mentioned. I feel like this has all the permissions I need 👍